File: chatwoot/ruby/src/AddNewAgentToTeamExample.rb
require "json"
require "chatwoot_client"

ChatwootClient.configure do |config|
    config.api_key["api_access_token"] = "USER_API_KEY"
end

add_new_agent_to_team_request = ChatwootClient::AddNewAgentToTeamRequest.new
add_new_agent_to_team_request.user_ids = [
]

begin
    response = ChatwootClient::TeamsApi.new.add_new_agent_to_team(
        0, # account_id
        0, # team_id
        add_new_agent_to_team_request, # data
    )

    p response
rescue ChatwootClient::ApiError => e
    puts "Exception when calling TeamsApi#add_new_agent_to_team: #{e}"
end

File: chatwoot/python/src/AddNewAgentToTeamExample.py
import json
from datetime import date, datetime
from pprint import pprint

from chatwoot_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"userApiKey": "USER_API_KEY"},
)

with ApiClient(configuration) as api_client:
    add_new_agent_to_team_request = models.AddNewAgentToTeamRequest(
        user_ids=[
        ],
    )

    try:
        response = api.TeamsApi(api_client).add_new_agent_to_team(
            account_id=0,
            team_id=0,
            data=add_new_agent_to_team_request,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling TeamsApi#add_new_agent_to_team: %s\n" % e)
